tlvc/init/main.c

```diff
@@ -334,10 +345,10 @@ static void INITPROC parse_parms(int cnt, char *line, int *nums, int base)
 		m = l;
 		while ((*l) && (*l != ',')) l++;
 		c = *l;		/* ensure robust eol handling */
-		if (l > m) {
+		if (l > m && *m != '-') {
```

Comment:
> Thus -1 is permitted and converted to zero in order to avoid adding support for negative numbers in init/main.c. 

`simple_strtol` does handle (or at least it's coded to supposed to handle) negative numbers. If this isn't working, perhaps there's a bug in it? I can chase that down if required.

Reply:
Interesting - I'm guilty in not testing this, instead assuming based on some vague (unreliable) memory that negative #s weren't supported.  Time to simplify even more.
Thanks.